chsdi/__init__.py

```diff
@@ -103,40 +103,10 @@ def datetime_adapter(obj, request):
     config.add_route('historicalmaps', '/historicalmaps/viewer.html')
     config.add_route('checker', '/checker')
     config.add_route('checker_dev', '/checker_dev')
-    config.add_route('backend_checker', '/backend_checker')
     config.add_route('downloadkml', '/downloadkml')
     config.add_route('translations', '/rest/services/translations')
     config.add_route('wmsconfig', '/rest/services/wmsconfig')
 
-    # kml files
-    config.add_route('files_collection', '/files', request_method=('GET', 'POST', 'DELETE'))
-    add_cors_route(config,
-                   '/files',
-                   'files_collection',
-                   headers={'Access-Control-Allow-Credentials': 'true'},
-                   methods=['GET', 'POST', 'DELETE'])
-    config.add_route('files', '/files/{id}', request_method=('GET', 'POST', 'DELETE'))
-    add_cors_route(config,
-                   '/files/{id}',
-                   'files',
-                   headers={'Access-Control-Allow-Credentials': 'true'},
-                   methods=['GET', 'POST', 'DELETE'])
-
-    # glstyles json files
-    config.add_route('glstyles_collection', '/gl-styles', request_method=('GET', 'POST', 'DELETE'))
-    add_cors_route(config,
-                   '/gl-styles',
-                   'glstyles_collection',
-                   headers={'Access-Control-Allow-Credentials': 'true'},
-                   methods=['GET', 'POST', 'DELETE'])
-    config.add_route('glstyles', '/gl-styles/{id}', request_method=('GET', 'POST', 'DELETE'))
-    add_cors_route(config,
-                   '/gl-styles/{id}',
-                   'glstyles',
-                   headers={'Access-Control-Allow-Credentials': 'true'},
-                   methods=['GET', 'POST', 'DELETE'])
-
-    config.add_route('adminkml', '/admin/kml')
```

Comment:
Do we also remove the admin interface to KML?

Reply:
Yup, we can remove that